jobs/enthought_web-developer.html

```diff
@@ -0,0 +1,41 @@
+---
+title: Full Stack Web Developer 
+company: Enthought
+url: http://www.enthought.com/company/careers/
+location: Cambridge, UK
```

Comment:
You can give a specific address (even with a postcode if you like) - that will allow candidates to find you on our map.